internal/cltest/cltest.go

```diff
@@ -135,13 +136,15 @@ func NewApplication() (*TestApplication, func()) {
 // NewApplicationWithConfig creates a New TestApplication with specified test config
 func NewApplicationWithConfig(tc *TestConfig) (*TestApplication, func()) {
 	app := services.NewApplication(tc.Config).(*services.ChainlinkApplication)
-	server := newApiServer(app)
-	tc.Config.ClientNodeURL = server.URL
+	apiServer := newApiServer(app)
+	guiServer := newGuiServer(app)
+	tc.Config.ClientNodeURL = apiServer.URL
 	app.Store.Config = tc.Config
 	ethMock := MockEthOnStore(app.Store)
 	ta := &TestApplication{
 		ChainlinkApplication: app,
-		Server:               server,
+		ApiServer:            apiServer,
+		GuiServer:            guiServer,
```

Comment:
Could we leave the GUI server off by default and only attach it for tests that need it?

Reply:
How do you feel about 2 types of application? e.g. `TestAPIApplication` & `TestGUIApplication`?

Reply:
Sounds good to me.